### ISSUE TITLE
UI Bug: NavBar not dynamic: smaller screens need to zoom out to view login/signup

### ISSUE THREAD

**Issue by forrestfiller** [ryanab/fileshare#8]
Bug
---

 - Bug found by Dan when he viewed the Homepage on his 11" monitor with Chome, he had to reduce the zoom to 90% in order to bring the login/signup function back on screen.

    - I was hardcoding the nav item over the right, so will need a dynamic spacing solution that works off of the nav items to the left. I didn't notice the issue as my monitor is larger, 13".

**Comment by jdrichardstech** [comment]
in the ui will media queries need to be set with css to address that bug?

**Comment by forrestfiller** [comment]
@jdrichardstech yeah, I guess so, I could start going in there, I was just hoping to fix with inline before going to the "dark place". I'm calling it for tonight, I'll jump back on the horse tomorrow.
